src/lib.rs

```diff
@@ -617,6 +618,22 @@ pub trait Itertools : Iterator {
         adaptors::step(self, n)
     }
 
+    /// Convert each item of the iterator using the `Into` trait.
+    ///
+    /// # Examples
```

Comment:
Oh by the way this trait doesn't use headers right now. Traditionally, to not overcrowd the docs, the headings used to be too large compared with the methods..